.gitattributes

```diff
@@ -0,0 +1 @@
+example_files/*.nc filter=lfs diff=lfs merge=lfs -text
```

Comment:
@thogar-computer Not completely sure tbh. It is generated automatically once you set git lfs to track a certain set of files. Everyone else seems to also use these flags too so I  think they're fine. 

I know the '-text' part treats the gitlfs files as binary files instead of text, so that way there is no line-ending conversion shenanigans happening that could mess with stuff.